src/app/community/community.component.ts

```diff
@@ -51,7 +55,10 @@ export class CommunityComponent implements OnInit {
   }
 
   ngOnInit() {
+    var urlFragment = this.router.url.split('/')[2];
+    if(urlFragment != "" && urlFragment != undefined){
```

Comment:
You could shorten it to `if (!!urlFragment) {` since it's a truthy value. More info: https://basarat.gitbooks.io/typescript/docs/tips/truthy.html